### PR TITLE
chapter08: Add prometheus instance

### PR DESCRIPTION
**Pull request by kintoandar**

### CODE REVIEW

File: chapter08/provision/cmd.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+source /vagrant/utils/helpers.sh
+
+check_requirements git
+
+if ! check_cache_bin "tsdb" || ! check_cache_bin "pprof" ; then
+  apt-get update
+  DEBIAN_FRONTEND=noninteractive apt-get install -y golang-go graphviz
+
+  go get github.com/prometheus/tsdb/cmd/tsdb
+  go build github.com/prometheus/tsdb/cmd/tsdb
```
Comment (pharaujo) [inline chapter08/provision/cmd.sh]: `go get` already builds the binary. We can probably drop this

File: chapter08/provision/cmd.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  install -m 0755 ./tsdb /vagrant/cache/
```
Comment (pharaujo) [inline chapter08/provision/cmd.sh]: If we just do `go get`, the binary will be located at `$HOME/go/bin/tsdb`

File: chapter08/provision/cmd.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+  go get -u github.com/google/pprof
```
Comment (pharaujo) [inline chapter08/provision/cmd.sh]: why the `-u`?

Comment (kintoandar) [reply chapter08/provision/cmd.sh]: No need

**Pull request merged**